backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -284,159 +284,100 @@ async def get_speaker_service_status():
-            return {
-                "message": "Memory configuration updated and reloaded successfully",
-                "config_path": config_path,
-                "backup_created": os.path.exists(backup_path),
-                "status": "success"
-            }
-        else:
-            return JSONResponse(
-                status_code=500, content={"error": "Configuration saved but reload failed"}
-            )
-        
+            return JSONResponse(status_code=400, content={"error": f"Invalid YAML syntax: {str(e)}"})
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)